apps/toi-arbeidsgiver-notifikasjon/src/main/kotlin/no/nav/arbeidsgiver/toi/arbeidsgiver/notifikasjon/templates.kt

```diff
@@ -11,68 +11,59 @@ const val TEKST = "{tekst}"
 const val AVSENDER = "{avsender}"
 
 val epostTemplate = """
-    <html>
+    <!DOCTYPE html>
+    <html lang='no'>
         <head>
-            <meta http-equiv='Content-Type' content='text/html; charset=UTF-8'/>
+            <meta http-equiv='Content-Type' content='text/html; charset=UTF-8' />
             <title id='tittel'>$TITTEL</title>
         </head>
-        <body style='font-family: sans-serif; padding:40px 20px; color: #262626'>
-            <div>
-                <h1 style='font-size: 1.75rem; font-weight:bold;'>Hei.</h1>
-                <h2 style='font-weight: 500; font-size: 1rem;'>Vi har funnet nye kandidater for deg til stillingen: <b id='stillingstittel'>$TITTEL</b>.</h2>
-                <pre style='font-family: unset;' id='tekst'>$TEKST</pre>
+        <body style='font-family: sans-serif; padding: 40px 20px; color: #23262a; mso-line-height-rule: exactly;'>
+            <h1 style='font-size: 32px; font-weight: bold'>Hei.</h1>
+            <p>Vi har funnet nye kandidater for deg til stillingen: <b id='stillingstittel'>$TITTEL</b>.</p>
+            <p style='white-space: pre-wrap; margin-top: 32px; margin-bottom: 32px' id='tekst'>$TEKST</p>
```

Comment:
Gjorde denne til en `p` i stedet for `pre`. Var mye problemer med tekst og font i MS Outlook med `pre`.